### PR TITLE
[3. Attach a note] Validation

### PR DESCRIPTION
**Pull request by LinX64**

### CODE REVIEW

**Comment by LinX64** [comment]
This is a new branch form this PR: https://github.com/HabitatMap/AircastingAndroid/pull/638

**Comment by LinX64** [comment]
[[3. Attach a note] Validation](https://trello.com/c/zycdfEw0/1758-3-attach-a-note-validation)

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -405,6 +405,10 @@
     <string name="from">de</string>
     <string name="to">à</string>
     <string name="login_out_background">Déconnexion en cours en arrière-plan. Vous pouvez remplir les informations d\'identification</string>
+    <string name="notes_des_cant_be_empty">La description des notes ne peut pas être vide!</string>
+    <string name="dialog_header_txt">\"Aircasting\" souhaite accéder à la caméra</string>
+    <string name="dialog_helper_des">Cela vous permettra de joindre des photos à vos notes de session mobile.</string>
+    <string name="dont_allow_dialog_txt">Ne pas autoriser</string>
```
Comment (xameYeoh) [inline app/src/main/res/values-fr/strings.xml:411]: I think this "txt" is redundant, we already know this is a string resource, it cannot be anything except text

**Review by xameYeoh — APPROVED** [review]
Looks cool! I've left some naming comments. 
I think that these changes could be referenced on the card as a "commit attachment" - you could include them in the https://github.com/HabitatMap/AircastingAndroid/pull/638 pr as one commit, copy the link of the commit and attach it to the card. I've tried it once and I think it's really convenient, as it doesn't require to create a separate pr for little code if there is already a base pr opened

**Comment by LinX64** [comment]
I've provided the changes after code review in this PR: https://github.com/HabitatMap/AircastingAndroid/pull/667

**Pull request closed without merging**